di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/handlers/CoreStubHandler.java

```diff
@@ -138,6 +136,19 @@ private void setQuestions() {
                 return ViewHelper.render(modelMap, "search-results.mustache");
             };
 
+    // Used where sharedClaim is entered as raw JSON string from browser for DL CRI
+    public Route sendSharedClaim =
+            (Request request, Response response) -> {
+                var credentialIssuer =
+                        handlerHelper.findCredentialIssuer(
+                                Objects.requireNonNull(request.queryParams("cri")));
+                String queryString = request.queryParams("claimsText");
+                String context = request.queryParams("context");
+                SharedClaims sharedClaims = objectMapper.readValue(queryString, SharedClaims.class);
```

Comment:
Can we add a try catch round this object mapper? We want to log when the structure of the shared Claims is incorrect

Reply:
And can we also add a log when we successfully map the queryString to sharedClaims to say "Raw JSON in form input mapped to shared claims"